### ISSUE TITLE
Receiving GCP is invalid error when running Plan from Terraform Cloud

### ISSUE THREAD

**Issue by genecurtisgoldol** [terraform-google-modules/terraform-example-foundation#1357]
### TL;DR

I am running the deployment as described in the README-Terraform-Cloud.md. All works as expected until I get to step 26: (Optional) Run terraform plan to verify that state is configured correctly.  The plan immediately fails with the following error:

Operation failed: configuration for GCP is invalid: missing required value(s): GCP project number, GCP workload pool ID, GCP workload provider ID

### Expected behavior

Expected Terraform plan to run to completion and report "No Changes."

### Observed behavior

Right after the plan starts, I receive the error that GCP is invalid (See below)

To view this run in a browser, visit:
https://app.terraform.io/app/ma-foundations/0-shared/runs/run-vfAjLUN19zTouziB

Waiting for the plan to start...

Terraform v1.10.2
on linux_amd64
Operation failed: configuration for GCP is invalid: missing required value(s): GCP project number, GCP workload pool ID, GCP workload provider ID

### Terraform Configuration

```hcl
Configuration is identical to the terraofmr-example-foundation project, with changes as defined in the Terraform Cloud readme.
```

### Terraform Version

```sh
1.10.2.
```

### Additional information

no.

**Comment by genecurtisgoldol** [comment]
Bug found.  Actually user error.  The variable 

resource "tfe_variable" "gcp_workload_provider_name"

was incorrectly commented out. With that varaible added, terraform plan completes.
